sed/calibrator/energy.py

```diff
@@ -2200,22 +2211,20 @@ def residual(pars, time, data, binwidth, binning, energy_scale):
     ecalibdict["t0"] = result.params["t0"].value
     ecalibdict["E0"] = result.params["E0"].value
     ecalibdict["energy_scale"] = energy_scale
+    energy_offset = pfunc(-1 * ref_energy, pos[0])
+    ecalibdict["E0"] = -(energy_offset - vals[0])
```

Comment:
We build a partial function with offset and TOF as free parameters, and evaluate this at the TOF value (pos[0]) of the reference feature in the first trace, with given energy value (-ref_energy). This returns the difference of the calibration method to the expected value + the first bias voltage. This value minus the respective bias voltage is the negative offset to apply to obtain the correct energy value.

Reply:
I'd suggest to rename this partial function to also tof2ev as it will just use the local scope. 
Using pfunc is not fully explanatory.

Reply:
Added as issue